lib/block_view_message.go

```diff
@@ -1227,12 +975,6 @@ func (bav *UtxoView) _disconnectMessagingGroup(
 			utxoOpsForTxn[operationIndex].Type)
 	}
 
-	// Check that the transaction has the right TxnType.
```

Comment:
I actually like having this sanity check in there. I think it's okay to add it in.